<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BackupBandwidthSettingsWidget</class>
 <widget class="QWidget" name="BackupBandwidthSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>245</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="widgetLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>16</number>
     </property>
     <property name="leftMargin">
      <number>16</number>
     </property>
     <property name="topMargin">
      <number>16</number>
     </property>
     <property name="rightMargin">
      <number>16</number>
     </property>
     <property name="bottomMargin">
      <number>16</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="limitTypeLayout">
       <item>
        <widget class="QComboBox" name="limitTypeComboBox">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>No Limit</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Fixed</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Scheduled</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="noLimitPage">
        <layout class="QVBoxLayout" name="noLimitPageLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
        </layout>
       </widget>
       <widget class="QWidget" name="fixedLimitPage">
        <layout class="QVBoxLayout" name="fixedLimitPageLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="fixedLimitLabel">
             <property name="text">
              <string>Fix bandwidth to</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="fixedLimitSpinbox">
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>9999</number>
             </property>
             <property name="value">
              <number>15</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="fixedLimitSuffixLabel">
             <property name="text">
              <string>Mbit/s</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="scheduledLimitPage">
        <layout class="QVBoxLayout" name="scheduledLimitPageLayout" stretch="0,1">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>8</number>
           </property>
           <property name="leftMargin">
            <number>40</number>
           </property>
           <item>
            <widget class="nx::vms::client::desktop::ScheduleBrushSelectableButton" name="unlimitedButton">
             <property name="minimumSize">
              <size>
               <width>36</width>
               <height>36</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>36</width>
               <height>36</height>
              </size>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="unlimitedLabel">
             <property name="text">
              <string>Unlimited</string>
             </property>
             <property name="buddy">
              <cstring>unlimitedButton</cstring>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>8</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="nx::vms::client::desktop::ScheduleBrushSelectableButton" name="limitedButton">
             <property name="minimumSize">
              <size>
               <width>36</width>
               <height>36</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>36</width>
               <height>36</height>
              </size>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="scheduledLimitLabel">
             <property name="text">
              <string>Limited to</string>
             </property>
             <property name="buddy">
              <cstring>limitedButton</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="scheduledLimitSpinBoxGroup" native="true">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>0</height>
              </size>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="spacing">
               <number>8</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QSpinBox" name="scheduledLimitSpinBox">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>1000</number>
                </property>
                <property name="value">
                 <number>15</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="scheduledLimitSuffixLabel">
                <property name="text">
                 <string>Mbit/s</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>8</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="nx::vms::client::desktop::ScheduleBrushSelectableButton" name="noBackupButton">
             <property name="minimumSize">
              <size>
               <width>36</width>
               <height>36</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>36</width>
               <height>36</height>
              </size>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="noBackupLabel">
             <property name="text">
              <string>No backup</string>
             </property>
             <property name="buddy">
              <cstring>noBackupButton</cstring>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="scheduleGridWidgetLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="nx::vms::client::desktop::ScheduleGridWidget" name="scheduleGridWidget" native="true"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="nx::vms::client::desktop::MessageBar" name="messageBar" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>nx::vms::client::desktop::ScheduleGridWidget</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/resource_properties/schedule/schedule_grid_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::ScheduleBrushSelectableButton</class>
   <extends>QToolButton</extends>
   <header location="global">nx/vms/client/desktop/resource_properties/schedule/schedule_brush_selectable_button.h</header>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::MessageBar</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/common/widgets/control_bars.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
